### PR TITLE
[RHELC-1193] Remove find_highest_report_level workaround

### PR DESCRIPTION
**Pull request by pr-watson**
This PR removes the function find_highest_report_level and replaces the output.status field with a new status field from the convert2rhel json file. Subsequent tests for find_highest_report_level have been removed as well.

### CODE REVIEW

File: scripts/conversion_script.py [hunk 1]
```diff
@@ -57,7 +57,7 @@ def __init__(
         self.error = error  # true if the script wasn't able to finish, otherwise false
         self.message = message
         self.report = report
-        self.tasks_format_version = "1.0"
+        self.tasks_format_version = "1.1"
```
Comment (r0x0d) [inline scripts/conversion_script.py:60]: I don't think this needs a version bump, as nothing changed in the format

Comment (pr-watson) [reply scripts/conversion_script.py:60]: I'll revert this change

**Comment by andywaltlova** [comment]
Rebased the PR because we did substantial changes and moved blocks of code here and there in last couple of PRs :)

File: README.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ Structure of repository is following:
 ├── requirements.txt  # DEV requirements - tests & lint
 ├── schemas # All expected json outputs in the scripts stdouts
 |   |   ...
-│   └── preconversion_assessment_schema_1.0.json
+│   └── preconversion_assessment_schema_1.1.json
```
[the remark this answers is not in the document]
Comment (pr-watson) [reply README.md:15]: I can revert it, just figured the latest schema should be in the README

[the remark this answers is not in the document]
Comment (andywaltlova) [reply README.md:15]: Oh yeah! Sorry, I should stop working today :sweat_smile:  Everything is fine i nthat case, thanks for the response!

**Review by r0x0d — APPROVED** [review]

**Pull request merged**